canonical-web-and-design/canonical.com
# Results apply

## Done

- updated /careers/apply to dynamically to show roles selected from game results
- updated modal on the same page to show a list of all available roles and allow the user to change the roles they've selected

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/careers/start
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Select a few options, submit them, you'll be taken to /careers/results
- Check some of the jobs in the list, at the bottom of the page click "Apply Now"
- See that you are taken to /careers/apply, and the jobs you chose are listed above the form
- Click 'Add more roles', select some roles and click "Add Roles" to close the modal
- See that the list of roles has been updated


## Issue / Card

Fixes #129
